server.py

```diff
@@ -1,36 +1,46 @@
 from flask import Flask, request, jsonify, g
 from cryptography import exceptions
-from logging import Formatter
 from decrypter import Decrypter
+from structlog import wrap_logger
 import binascii
 import settings
 import logging
-import logging.handlers
+import sys
+
 
 app = Flask(__name__)
 
+logging.basicConfig(stream=sys.stdout, level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
+logger = wrap_logger(
+    logging.getLogger(__name__)
+)
+logger.debug("START")
+
 
 def get_decrypter():
     # Sets up a single decrypter throughout app.
-
     decrypter = getattr(g, '_decrypter', None)
-
     if decrypter is None:
         try:
             decrypter = g._decrypter = Decrypter()
         except Exception as e:
-            app.logger.error("Decrypter failed to start")
-            app.logger.error(repr(e))
+            logger.error("Decrypter failed to start")
+            logger.error(repr(e))
```

Comment:
We want the exception string associated with the log line, so something like `logger.error("Decrypter failed to start", exception=repr(e))` would be better.